bin/update-patterns

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
```

Comment:
needs a description in a comment at the top, what it adds and where it does come from (Github source repository)